### DISCUSSION TITLE
mezzanine website not working

### DISCUSSION THREAD

**Discussion by davidjeba** [stephenmcd/mezzanine#2037] [Q&A]
is there a marketing website for mezzanine cms ?

**Comment by kenbolton** [comment]
You are soaking in it! (Apologies for the 80s American pop-culture reference: https://www.youtube.com/watch?v=dzmTtusvjR4)

There was a marketing site at mezzanine.jupo.org but it now redirects to the README here.

I do not believe that there is a great thirst to build and maintain a marketing and demo site as in the olden days. The target audience of Mezz is expected to have some facility with git and github, and the developers are generally occupied with building real things.

**Comment by henri-hulski** [comment]
See https://github.com/stephenmcd/mezzanine/issues/2030#issuecomment-1149627343

**Reply by kenbolton** [reply]
Maybe it is a false distinction, but the solution you propose is for the documentation, whereas this question and the old mezzanine.jupo.org are about marketing. 

Stephen has already been extremely generous with his time and resources over the life of this project. Relying on him beyond the existing redirect seems, to me, unnecessary.

**Reply by henri-hulski** [reply]
Yeah I'm speaking about documentation.
On @stephenmcd 's side this would only be a one time action to activate gh-pages and add the CNAME records to mezzanine.jupo.org and cartridge.jupo.org as he is the only one with access rights. And he agreed but was not sure if it's possible so I did some more research and it should be possible as explained above.

**Reply by henri-hulski** [reply]
I wasn't aware that mezzanine.jupo.org was also a marketing site.

**Reply by kenbolton** [reply]
mezzanine.jupo.org was marketing with documentation at mezzanine.jupo.org/docs/ (or maybe at /documentation/). not a big deal, just trying to provide some historical context.

**Reply by henri-hulski** [reply]
Yep I think I remember now how it looked like.
